test/util_test.go

```diff
@@ -92,8 +147,83 @@ func startDqlite(ctx context.Context, tb testing.TB) endpoint.Config {
 		app.Close()
 	})
 
-	return endpoint.Config{
-		Listener: fmt.Sprintf("unix://%s/listen.sock", dir),
+	db, err := app.Open(ctx, "k8s")
+	if err != nil {
+		tb.Fatal(err)
+	}
+
+	return &endpoint.Config{
+		Listener: fmt.Sprintf("unix://%s/kine.sock", dir),
 		Endpoint: fmt.Sprintf("dqlite://k8s?driver-name=%s", app.Driver()),
+	}, db
+}
+
+func setupScenario(ctx context.Context, db *sql.DB, prefix string, numInsert, numUpdates, numDeletes int) error {
+	t, err := db.BeginTx(ctx, nil)
+	if err != nil {
+		return err
+	}
+	defer t.Rollback()
+
+	insertManyQuery := `
+WITH RECURSIVE gen_id AS(
+	SELECT 1 AS id
+
+	UNION ALL
+
+	SELECT id + 1
+	FROM gen_id
+	WHERE id < ?
+), revision AS(
+	SELECT COALESCE(MAX(id), 0) AS base
+	FROM kine
+)
+INSERT INTO kine(
```

Comment:
This may be an existing convention, but I think it could be nice to put some newlines between the separate parts of these queries

Reply:
It is an existing convention not to.